### PR TITLE
Mention environment reward in the observations.rst

### PR DESCRIPTION
**Pull request by Gamenot**

### CODE REVIEW

File: docs/sim/observations.rst [hunk 1]
```diff
@@ -71,6 +71,10 @@ See implemention in :class:`smarts.core.sensors`
 
 Then, you can choose the observations needed through :class:`smarts.core.agent_interface.AgentInterface` and process these raw observations through :class:`smarts.core.observation_adapter`.
 
+=======
+Rewards
+=======
+The reward from smarts is given by a calculation in the `TripMeterSensor`. This is 0 or values `< -0.5 | 0.5 <` relating to distance traveled in meters when a vehicle has gone at least 0.5 meters since the last reward. `env_reward` from the smarts environment directly use this.
```
Comment (liamchzh) [inline docs/sim/observations.rst:77]: How does this `< -0.5 | 0.5 <` mean here?

Comment (JVillella) [reply docs/sim/observations.rst:77]: Minor: Do we need to mention TripMeterSensor? It's not extensible to the user unless they modify the source. I saw these docs as public API docs.

This is hard to understand as Liam said,

>This is 0 or values `< -0.5 | 0.5 <`

**Review by JVillella — CHANGES_REQUESTED** [review]
Few minor comments

File: docs/sim/observations.rst (hunk 1 above)
Comment (Gamenot) [reply docs/sim/observations.rst]: I have clarified it.

Comment (liamchzh) [reply docs/sim/observations.rst]: I am not sure whether we need to give the exact range in this `Rewards` explanation. 

But I am fine with both.

**Review by liamchzh — APPROVED** [review]

File: docs/sim/observations.rst (hunk 1 above)
Comment (Gamenot) [reply docs/sim/observations.rst]: I had a question about what the reward was from someone doing research so I would say it is better to.

**Review by aurorahcx — APPROVED** [review]

**Review by Adaickalavan — APPROVED** [review]

**Pull request merged**